docs/developer-docs/latest/developer-resources/data-management.md

```diff
@@ -197,3 +260,96 @@ npm strapi import -f export_20221213105643.tar.gz.enc --key my-encryption-key
+
+#### Example of the `--force` option
+
+<br/>
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi import -f export_20221213105643.tar.gz.enc --force --key my-encryption-key
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi import -f export_20221213105643.tar.gz.enc --force --key my-encryption-key
+```
+
+</code-block>
+</code-group>
+
+### Exclude data types during import
+
+The default `strapi import` command imports your content (entities and relations), files (assets), project configuration, and schemas. The `--exclude` option allows you to exclude content, files, and the project configuration by passing these items in a comma-separated string with no spaces between the types. You can't exclude the schemas, as schema matching is used for `strapi import`.
+
+::: warning
+Any types excluded from the import will be deleted in your target instance. For example, if you exclude `config` the project configuration in your target instance will be deleted.
+:::
+
+:::note
+Media such as images consist of the file (asset) and the entity in the database. If you use the `--exclude` flag to remove assets, the database records are still included, and could render as broken links.
+:::
+
+#### Example: exclude assets from an import
+
+<br/>
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi import -f export_20221213105643.tar.gz.enc --exclude files
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi import -f export_20221213105643.tar.gz.enc --exclude files
+```
+
+</code-block>
+</code-group>
+
+### Include only specified data types during import
+
+The default `strapi import` command imports your content (entities and relations), files (assets), project configuration, and schemas. The `--only` option allows you to export only the listed items by passing a comma-separated string  with no spaces between the types. The available values are `content`, `files`, and `config`. Schemas are always imported, as schema matching is used for `strapi import`.
+
+:::note
+Media such as images consist of the file (asset) and the entity in the database. If you use the `--only` flag to import `content` the asset database records are still included, and could render as broken links.
+:::
+
+#### Example: import only the project configuration
+
+<br/>
+<code-group>
+<code-block title="YARN">
+
+```bash
+yarn strapi import -f export_20221213105643.tar.gz.enc --only config
+```
+
+</code-block>
+
+<code-block title="NPM">
+
+```bash
+npm strapi import -f export_20221213105643.tar.gz.enc --only config
+```
+
+</code-block>
+</code-group>
+
+:::note
+The `--exclude` option and `--only` option cannot be used together.
+:::
```

Comment:
They technically can, it just doesn't do anything. We only block using the exact same option for both, eg, `--only files --exclude files` will be an error, `--only content --exclude files` will run and only include content, so the `--exclude` in that case is useless but it won't throw an error

Reply:
OK thanks, in that case I think it is safe to just remove the callout.